sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -263,7 +263,7 @@ def __init__(
             **kwargs,
         )
 
-        self._rp_service_client = ServiceClient122022Preview(
+        self._rp_service_client = ServiceClient022023Preview(
```

Comment:
Think you may have missed this one, can we keep ServiceClient122022Preview here please? It is used by WorkspaceOperations and WorkspaceOutboundRuleOperations.